arcade/arcade/cli/utils.py

```diff
@@ -671,6 +671,7 @@ def handle_user_command(
             toolkit=None,
             tool=None,
             host=host,
+            local=False,
```

Comment:
What if I am actually chatting with localhost? e.g. `arcade chat -h`

Reply:
In this case, the `host=localhost`.

`local`: display tools in the local environment (never used for arcade chat)
`host=localhost`: display tools that the local actor is serving